### DISCUSSION TITLE
Can I make the row of the week scrollable?

### DISCUSSION THREAD

**Discussion by marcogian2001** [hi-manshu/Kalendar#35] [Ideas]
Hi there is any way to make the row scrollable? 
Thanks

**Comment by gsrathoreniks** [comment]
Negative, You've option to move to next or previous months using the corresponding
arrow icons.

**Comment by hi-manshu** [comment]
Hey there,
We plan to add it in future but not now! We will ping here again when we add that

**Comment by hi-manshu** [comment]
Hey its done. Please check the edlos package in 1.3.1
